### ISSUE TITLE
status line counter

### ISSUE THREAD

**Issue by alterratz** [t9md/atom-quick-highlight#21]
after quick-highlight:clear the counter in the status line still remains.
However the "natural" behavior would be to hide it when there is no selection.

**Comment by t9md** [comment]
yes, agree. I just soo lazy to fix it, I noticed it just after released current version.
